pkg/plugin/plugin.go

```diff
@@ -247,6 +265,28 @@ func (p *plugin) CreateOrUpdate(ctx context.Context, cs *api.OpenShiftManagedClu
 	return p.createOrUpdateExt(ctx, cs, updateTypeCreate, deployFn, false)
 }
 
+func (p *plugin) checkIfClusterWillRefresh(ctx context.Context, cs *api.OpenShiftManagedCluster) error {
+	if cs.Properties.RefreshCluster == nil || (cs.Properties.RefreshCluster != nil && *cs.Properties.RefreshCluster) {
```

Comment:
nit - this reduces to `if cs.Properties.RefreshCluster == nil || *cs.Properties.RefreshCluster` ("if it's nil, or it's set to true")